packages/lumx-react/src/components/list/List.tsx

```diff
@@ -21,6 +21,8 @@ export interface ListProps extends GenericProps {
     isClickable?: boolean;
     /** Item padding size. */
     itemPadding?: Extract<Size, 'big' | 'huge'>;
+    /** Tab index of the list. Default to -1 */
+    tabIndex?: -1 | 0;
```

Comment:
It's technically a breaking change to reduce the tabIndex to `0 | -1`
```suggestion
    tabIndex?: number;
```

Reply:
True, will change